db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_object_store_containers, :type, :string
```

Comment:
Yes, I think a multi column index would be necessary here

If we were to lookup by `type` we'd also lookup by what other fields?